src/patternfly/components/Panel/panel.scss

```
@@ -7,6 +7,7 @@
   --#{$panel}--ZIndex: auto;
   --#{$panel}--BackgroundColor: var(--pf-t--global--background--color--primary--default);
   --#{$panel}--BoxShadow: none;
+  --#{$panel}--BorderRadius: var(--pf-t--global--spacer--md);
```

if we want to match menu, it uses the small border radius

```suggestion
  --#{$panel}--BorderRadius: var(--pf-t--global--border--radius--small);
```

---

Oop yeah this should be the border--radius token. @lboehling do we want to match Menu or keep the medium border radius per discussion during working session?

---

i agree with @mcoker's points above! i'm down to match this with menu!